man/GGIR.Rd

```diff
@@ -831,12 +831,26 @@ GGIR(mode = 1:5,
         0.3 indicates that at least 30 percent of the time should be valid.}
         
       \item{segmentDAYSPTcrit.part5}{
-        Numeric vector or length 2 (default = c(0, 0)).
+        Numeric vector or length 2 (default = c(0, 0.9)).
         Inclusion criteria for the proportion of the segment that should be 
         classified as day (awake) and spt (sleep period time) to be considered 
-        valid. Usually, one of the two numbers is 0, and the other defines the 
-        proportion of the segment that should be classified as day or spt.}
-
+        valid. One of the two numbers should be 0, and the other defines the 
+        proportion of the segment that should be classified as day or spt, respectively.
+        The default setting would focus on waking hour
```

Comment:
I think the default should be the other way around to focus on waking hour, right? the first number indicates the proportion of the segment that should be classified as awake to be included, the second number indicates the proportion of the segment that should be classified as sleep to be included. 

I just checked the default settings and I only get "sleep" segments in the clean version of the daysummary_Segments file, so maybe we need to change the default or change the documentation here.

Reply:
Yes, you are right. I agree that focus on PA is most logical as default.